libs/parallel_utils.py

```diff
@@ -36,7 +36,7 @@ def parallel_map(func: Callable[[T], R], iterable: Iterable[T]) -> Iterable[R]:
         # But that might not be enough. Also make sure we don't spawn more than 32 processes (the
         # build machine is 96-core)
         processes = min(os.cpu_count(), 32)
-        with multiprocessing.Pool(maxtasksperchild=1, processes=processes) as pool:
+        with get_context("spawn").Pool(maxtasksperchild=1, processes=processes) as pool:
```

Comment:
I had issues using the `fork` method (both locally and on the runner) where the program would hang indefinitely after completing all the parallel processes, but using `spawn` fixed those issues. 

I see the TODOs and our issue with using `spawn` with pandarallel (https://github.com/nalepae/pandarallel/issues/72), but I _think_ because using `get_context()` only sets the method inside the context manager this should be compatible (and I haven't run into any issues so far.)